src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,48 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../../Types/posts';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface Props {
+  post: Post
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {(comments && comments.length > 0)
```

Comment:
```suggestion
      {comments.length > 0
```
Filter method is used to find the comments , if there are no comments it will be an empty array so it will always exist , checking the length is enough. .